### DISCUSSION TITLE
lua_scripts/HardcoreMode.lua:99: attempt to concatenate global 'playerName' (a nil value)

### DISCUSSION THREAD

**Discussion by mpluswow** [stylo019/SkyWall-HardCore-Mode-3.3.5-WoW#4] [Q&A]
Anyone could tell me why im getting this error while trying to use hardcore mode ?
Is it due to me having GM account ?

lua_scripts/HardcoreMode.lua:99: attempt to concatenate global 'playerName' (a nil value)

**Comment by stylo019** [comment]
Maybi it will help 
https://github.com/stylo019/SkyWall-HardCore-Mode-3.3.5-WoW/issues/2
